tests/Issues/Issue7495/Fixture/fixture.php.inc

```diff
@@ -44,7 +44,6 @@ class Fixture
     private Registry $coreRegistry;
 
     public function __construct(
-        UrlInterface $urlInterface,
```

Comment:
this also need to be reverted as well, as next param is not removed, only property fetch that to be removed.

Reply:
https://github.com/rectorphp/rector-src/pull/4016